app/src/main/res/values/strings.xml

```diff
@@ -336,7 +336,7 @@
     <string name="space_units">" units"</string>
     <string name="units">units</string>
     <string name="carbs">carbs</string>
-    <string name="when">when</string>
+    <string name="when">time</string>
```

Comment:
"time" exists as a resource string, already: https://github.com/NightscoutFoundation/xDrip/blob/42f0f837bd31a2875fd32646b9252dcbb27985ba/app/src/main/res/values/strings.xml#L478

To use it, adapt https://github.com/NightscoutFoundation/xDrip/blob/38fee03be56da6ce9cbe90e08dbe8e4b651c6edc/app/src/main/java/com/eveningoutpost/dexdrip/PhoneKeypadInputActivity.java#L508.

Reply:
Yes, "when (HHMM)" is good. I think that would solve the confusion.